src/character/EvercraftCharacter.java

```diff
@@ -30,9 +39,14 @@ public Integer getArmorClass() {
 	}
 
 	public Integer getHitPoints() {
-		return hitPoints;
+		return hitPoints + levelHitPointIncrease();
```

Comment:
Perhaps we should move this logic to a modifier class. The level hit point increase can be applied at the same time the "constitution" ability increase is applied. That way all modification to the base character attribute occurs in a single place (This might be confusing... but will be more easy to understand when the abilities PR is finished)